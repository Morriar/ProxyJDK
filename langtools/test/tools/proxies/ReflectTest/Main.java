/*
 * @test
 * @summary reflect test
 * @author Alexandre Terrasa
 *
 * @compile Main.java
 */

package TMPProxyTests.ReflectTest;

import java.io.Serializable;
import java.lang.reflect.Field;

public class Main {

	/**
	 * @param args
	 */
	public static void main(String[] args) throws NoSuchFieldException {
		
		Serializable prxy = proxy Serializable("foo");
		
		Class pClass = prxy.getClass();
		
		System.out.println(pClass);
		
		for(Field d : pClass.getFields()) {
			System.out.println(d);
		}
	}

}
