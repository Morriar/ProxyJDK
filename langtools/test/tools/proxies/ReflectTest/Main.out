class TMPProxyTests.java.io.ProxySerializable